bitwarden_license/src/Sso/Utilities/ClaimsExtensions.cs

```diff
@@ -9,7 +9,7 @@ namespace Bit.Sso.Utilities
     public static class ClaimsExtensions
     {
         private static readonly Regex _normalizeTextRegEx =
-            new Regex(@"[^a-zA-Z]", RegexOptions.CultureInvariant | RegexOptions.Singleline);
+            new Regex(@"[^a-zA-Z]", RegexOptions.CultureInvariant | RegexOptions.Singleline | RegexOptions.Compiled);
```

Comment:
This seems like a blind or premature optimization here. Generally this isn't necessary given the low number of Regex uses we have, and the 15 or so supported LRU cache entries would include this and other similar use of Regex to reduce redundant evaluation costs, w/o tripling to quintupling compilation IL time for producing these compiled expressions on startup (especially as in a fully auto-scaled k8s environment, apps could be starting up and being torn down frequently).

Reply:
Sounds good, I would agree this might not be good for k8s environment. And these pros/cons might all be moot if they release a source generator for compiled regex in .NET 7, which would give us the increased speed and no startup cost.